tests/storages_tests/test_cached_storage.py

```diff
@@ -77,65 +54,44 @@ def test_uncached_set() -> None:
     storage = _CachedStorage(base_storage)
     study_id = storage.create_new_study("test-study")
 
+    trial_id = storage.create_new_trial(study_id)
```

Comment:
Sorry, this comment doesn't apply to this line specifically but rather `test_uncached_set`, but the docs of this functions should be updated to reflect the new flushing behavior.

Reply:
Thank you for the comments. 

Is this fixed as you intended?
https://github.com/optuna/optuna/pull/3120/files#diff-4c5bf6b243e65dfd008619eaf78d0da4f88ee2c36948cd40fd74dc1a4ebc60e3R48-R50

Reply:
Looks good. We might even write `The CachedStorage flushes any modification of trials to a persistent storage immediately.` which might be clearer. What do you think?

Reply:
OK. I will modify it. Thank you

Reply:
Thanks for the prompt fix, let me resolve this thread. 🙇